WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.kt

```diff
@@ -1051,6 +1060,73 @@ class MySiteFragment : Fragment(),
         }
     }
 
+    @Subscribe(sticky = true, threadMode = ThreadMode.MAIN)
+    fun onEventMainThread(event: StorySaveResult) {
+        EventBus.getDefault().removeStickyEvent(event)
```

Comment:
Another nitpick, I think we can leave this, but since we are trying to lessen the usage of these `static` calls directly, we can use the `EventBusWrapper`. Wdyt?

Reply:
given this is a Fragment I don't think there's much of a difference, but I may be missing something. That said, we can definitely give it another round for alpha2 when we change focus more on testability, sounds good?

Reply:
Yes, sounds good. Yes, this fragment won't be tested so we can leave this as is and look at it later. Thanks for the comment 🙏